cloud/linode/loadbalancers.go

```diff
@@ -601,12 +601,20 @@ func (l *loadbalancers) buildLoadBalancerRequest(ctx context.Context, clusterNam
 	return l.createNodeBalancer(ctx, clusterName, service, configs)
 }
 
+func trimString(s string, maxLen int) string {
+	if len(s) > maxLen {
+		return s[:maxLen]
+	}
+	return s
+}
+
 func (l *loadbalancers) buildNodeBalancerNodeCreateOptions(node *v1.Node, nodePort int32) linodego.NodeBalancerNodeCreateOptions {
 	return linodego.NodeBalancerNodeCreateOptions{
 		Address: fmt.Sprintf("%v:%v", getNodeInternalIP(node), nodePort),
-		Label:   node.Name,
-		Mode:    "accept",
-		Weight:  100,
+		// NodeBalancer backends must be 3-32 chars in length
+		Label:  trimString(node.Name, 32),
```

Comment:
might be dumb but the error says `Length must be 3-32 characters`... for safety sake should we pad the name if it's < 3 chars?

Reply:
Makes sense. There do not appear to be restrictions for a minimum length of a node's name other than adhering to the spec for a DNS subdomain as outlined by RFC 1123 ([source](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)). I guess it's valid to have a k8s node with a 1 character name?

I'll update to pad if needed.